src/test/fuzz/util/CMakeLists.txt

```diff
@@ -2,22 +2,61 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or https://opensource.org/license/mit/.
 
-add_library(test_fuzz STATIC EXCLUDE_FROM_ALL
-  descriptor.cpp
-  mempool.cpp
-  net.cpp
-  ../fuzz.cpp
-  ../util.cpp
-)
-
-target_link_libraries(test_fuzz
-  PRIVATE
-    core_interface
-    test_util
-    bitcoin_node
-    Boost::headers
-)
-
-if(NOT FUZZ_BINARY_LINKS_WITHOUT_MAIN_FUNCTION)
-  target_compile_definitions(test_fuzz PRIVATE PROVIDE_FUZZ_MAIN_FUNCTION)
+if(BUILD_INDIVIDUAL_FUZZ_BINARIES)
+  # bash command produces list of harnesses: <harness name> <source file>
+  execute_process(
```

Comment:
Currently an individual `test_fuzz_*` lib and `fuzz_*` binary is produced for each harness. It's kind of ugly to duplicate this loop but I'm not sure to avoid it. Another loop would likely need to be added for the wallet harnesses as well.

@hebasto @fanquake @maflcko  Any ideas?